docs/quick-start/troubleshooting.md

```diff
@@ -99,72 +106,134 @@ description: Stuck on your ExpressLRS setup? Let us help you with that! Here's s
 
     See [MSP Section](pre-1stflight.md#msp) for more info.
 
-### <span class="custom-heading" data-id="11">My Blackbox Log has stepping on the RC trace</span>
+### <span class="custom-heading" data-id="12">I cannot change the Switch Mode!</span>
 
-??? Note "My Blackbox Log has stepping on the RC trace"
-    Make sure you have disabled ADC Filter in your Radio Hardware settings.
+??? Note "I cannot change the Switch Mode!"
+    Changing switch modes requires that the TX module and Receiver aren't connected to each other (no C in the top right corner of the Lua Script). Power off the receiver first, change the Switch Mode from the Lua script, then power up the receiver. The Switch Mode should then apply.
 
-## Flashing/Updating
+### <span class="custom-heading" data-id="13">I cannot change my Packet Rate to F1000!</span>
+
+??? Note "I cannot change my Packet Rate to F1000!"
+    F1000 requires higher than 400K baud rates. First update your baud rate setting either in the Model Setup menu or in the System Menu -> Hardware, reboot your radio to ensure that the baud rate setting got applied and then change the Packet Rate.
 
-!!! Tip
-    Individual hardware Flashing and Updating guide can be found by using the menu on the left sidebar.
+    Do note that not every handset is capable of higher than 400K baud rates. See the [Radio Setup Guide](tx-prep.md) for the details.
 
-    You can also use the Search Bar on top of the page to get to your device's Flashing Guide faster.
+### <span class="custom-heading" data-id="14">I cannot change to Full Resolution Rates!</span>
 
-### <span class="custom-heading" data-id="12">Invalid serial RX configuration detected</span>
+??? Note "I cannot change to Full Resolution Rates!"
```

Comment:
It's fine that this is here, but do not remove it from the Lua Script How-to Page.
If possible, give it its own section.